plugins/otter-pro/inc/plugins/class-form-emails-storing.php

```diff
@@ -67,6 +71,13 @@ public function init() {
 		add_action( 'add_meta_boxes', array( $this, 'add_form_record_meta_box' ) );
 		add_action( 'admin_menu', array( $this, 'handle_admin_menu' ) );
 		add_action( 'save_post', array( $this, 'form_record_save_meta_box' ), 10, 2 );
+
+		add_filter( 'otter_form_record_confirm', array( $this, 'confirm_submission' ), 10, 2 );
+
+		add_action( 'transition_post_status', array( $this, 'apply_hooks_on_draft_transition' ), 10, 3 );
```

Comment:
Not needed but you can also use it like this to target specific status:  `add_action( "draft_to_publish", 'do_stuff` );